tests/integration/routes/test_static.py

```diff
@@ -2,10 +2,6 @@
 
 
 class TestStatic(IntegrationTestCase):
-    def test_contact(self):
```

Comment:
Is this test definitely not needed anymore?

Reply:
I don't think so because eq no longer will have a contact us page. We just link to the campaign websites one in the footer.